coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/utils/TrinoSqlCallTransformerUtil.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.utils;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.sql.type.SqlReturnTypeInference;
+import org.apache.calcite.sql.validate.SqlUserDefinedFunction;
+
+import com.linkedin.coral.com.google.common.base.CaseFormat;
+import com.linkedin.coral.com.google.common.base.Converter;
+import com.linkedin.coral.com.google.common.collect.ImmutableMultimap;
+import com.linkedin.coral.common.functions.Function;
+import com.linkedin.coral.common.transformers.SignatureBasedConditionSqlCallTransformer;
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
+
+
+/**
+ * This utility class provides the functionalities of creating SqlCallTransformers and related SqlOperators
+ * based on the functions registered in StaticHiveFunctionRegistry to perform the transformation from Coral
+ * to Trino on SqlNode layer
+ */
+public class TrinoSqlCallTransformerUtil {
```

Comment:
Can we consider removing this class?

Reply:
Yes, this class is removed. Some functions in it is not necessary and removed and three remaining functions related to `HIVE_STATIC_REGISTRY` is moved to `CoralToTrinoSqlCallTransformersUtil`